ci/scripts/utils/ci_utils.sh

```diff
@@ -155,3 +154,24 @@ function publish_logs() {
     fi
     echo "${URL}"
 }
+
+function cleanup_experiment() {
+
+    local EXPDIR="$1"
+    local pslot
+    local ARCDIR
+    local ATARDIR
+    local DATAROOT
+
+    EXPDIR="$1"
+    pslot=$(basename "${EXPDIR}")
+
+    # Use the Python utility to get the required variables
+    read -r ARCDIR ATARDIR DATAROOT < <("${HOMEgfs}/ci/scripts/utils/get_config_var.py" ARCDIR ATARDIR DATAROOT "${EXPDIR}") || true
+
+    rm -Rf "${DATAROOT:?}"
+    rm -Rf "${ARCDIR:?}"
+    rm -Rf "${ATARDIR:?}"
+    rm -Rf "${EXPDIR}/../COMROOT/${pslot}"
```

Comment:
You could also read `COMROOT` from `config.base`:

```suggestion
    read -r ARCDIR ATARDIR DATAROOT COMROOT < <("${HOMEgfs}/ci/scripts/utils/get_config_var.py" ARCDIR ATARDIR DATAROOT COMROOT "${EXPDIR}") || true

    rm -Rf "${DATAROOT:?}"
    rm -Rf "${ARCDIR:?}"
    rm -Rf "${ATARDIR:?}"
    rm -Rf "${COMROOT:?}"
```